tools/config/Tomb1Main_ConfigTool/Resources/Lang/es.json

```diff
@@ -0,0 +1,322 @@
+    },
+    "airbar_location": {
+      "Description": "Ubicación donde se muestra la barra de aire.",
+      "Title": "Ubicación de la barra de aire"
+    },
+    "airbar_showing_mode": {
+      "Description": "Cambia la forma en que se muestra la barra de aire.\n\n- Predeterminado: muestra la barra de aire solo en el agua y parpadea la barra de aire cuando el oxígeno de Lara está al 20% o menos (TombATI).\n\n- Solo intermitente: muestra la barra de aire *solo* cuando el oxígeno de Lara está al 20% o menos (para carreras de desafío).\n\n- Nunca: nunca mostrar la barra de aire (para carreras de desafío).\n\n- PS1: muestra la barra de aire en circunstancias predeterminadas sin parpadeo.",
+      "Title": "Comportamiento de la barra de aire"
+    },
+    "anisotropy_filter": {
+      "Description": "Mejora el filtrado de texturas a distancia.",
+      "Title": "Filtro anisotrópico"
+    },
+    "change_pierre_spawn": {
+      "Description": "Hace que un Pierre recién activado (fugitivo) reemplace a un Pierre ya existente (fugitivo).",
+      "Title": "Cambiar el comportamiento de generación de Pierre"
+    },
+    "disable_healing_between_levels": {
+      "Description": "Evita que Lara se cure entre recargas de nivel (para las carreras de desafío No Heal).",
+      "Title": "No curar al terminar el nivel"
+    },
+    "disable_magnums": {
+      "Description": "Elimina todas las pistolas magnum y la munición magnum del juego.",
+      "Title": "Quitar magnum"
+    },
+    "disable_medpacks": {
+      "Description": "Elimina todos los botiquines del juego (para carreras de desafío No Meds).",
+      "Title": "Quitar botiquines"
+    },
+    "disable_shotgun": {
+      "Description": "Elimina todos las escopetas y cartuchos de escopeta del juego.",
+      "Title": "Quitar escopeta"
+    },
+    "disable_trex_collision": {
+      "Description": "Elimina toda colisión con T-Rex al morir. \nEsto ayuda cuando el cuerpo del T-Rex bloquea el paso.",
+      "Title": "Eliminar la colisión de T-Rex muerto"
+    },
+    "disable_uzis": {
+      "Description": "Elimina todas las armas Uzi y la munición Uzi del juego.",
+      "Title": "Quitar Uzis"
+    },
+    "enable_3d_pickups": {
+      "Description": "Permite renderizar modelos 3D en lugar de los sprites para recoger artículos.",
+      "Title": "Recogidas 3D"
+    },
+    "enable_braid": {
+      "Description": "Habilita la trenza de Lara.",
+      "Title": "Trenza"
+    },
+    "enable_cheats": {
+      "Description": "Habilita varios trucos:\n- F11: mantén pulsado para curar a Lara.\n- Tecla F11 + CAMINAR: mantén presionada para lastimar a Lara.\n- L: finaliza inmediatamente el nivel.\n- I: da a Lara todas las armas; un impulso de munición y botiquines y todos los elementos de la trama para el nivel actual.\n- O: habilita el truco DOZY (nadar en el aire).\n     \n- Tecla CAMINAR: salir DOZY.\n     \n- Tecla ARMAS: abre la puerta más cercana (no funciona en ciertos lugares).",
```

Comment:
Similar issue with line breaks here, but also the final two lines should be indented as they are related to the DOZY line.
![cheats](https://user-images.githubusercontent.com/33758420/223535171-1950ff29-ee30-4657-83a6-3dbe27998b35.png)
![cheats_en](https://user-images.githubusercontent.com/33758420/223535187-7773965c-fd6c-442a-92ba-e1bba7e91051.png)